cli/Sources/TuistLoader/Models+ManifestMappers/RunAction+ManifestMapper.swift

```diff
@@ -58,6 +58,15 @@ extension XcodeGraph.RunAction {
 
         let launchStyle = XcodeGraph.LaunchStyle.from(manifest: manifest.launchStyle)
 
+        var appClipInvocationURL: URL?
+        if let appClipInvocationURLString = manifest.appClipInvocationURLString {
+            if let url = URL(string: appClipInvocationURLString) {
+                appClipInvocationURL = url
+            } else {
+                throw URLError(.badURL)
```

Comment:
Let's create a `LocalizedError` scoped to this mapping logic where we can add an `errorDescription` that's very explicit about the issue. If we raise `URLError(.badURL)`, it'll be hard to diagnose the error when it happens.